code/controllers/subsystem/bots.dm

```diff
@@ -1,22 +0,0 @@
-var/datum/subsystem/bots/SSbot
-
-/datum/subsystem/bots
-	name = "Bots"
-	priority = 8
-
-	var/list/processing = list()
-
-/datum/subsystem/bots/New()
-	NEW_SS_GLOBAL(SSbot)
-
-/datum/subsystem/bots/stat_entry(msg)
-	..("P:[processing.len]")
-
-/datum/subsystem/bots/fire()
-	var/seconds = wait * 0.1
-	for(var/thing in processing)
-		if(thing && !thing:gc_destroyed)
-			spawn(-1)
```

Comment:
this is important--bot code utilizes sleeps in it, thus the reason why each of these are spawned off like this---if you move bots over to be mobs and retain those sleeps, it's going to cause issues.